src/soroban_rpc.ts

```diff
@@ -28,6 +28,7 @@ export namespace SorobanRpc {
     lastModifiedLedgerSeq?: number;
     key: xdr.LedgerKey;
     val: xdr.LedgerEntryData;
+    expiration?: xdr.ExpirationEntry;
```

Comment:
should we just return the sequence number? or is there more in the entry that might be useful to users?

Reply:
yeah, good point, I wondered about the expiration model aspect, how much this should be a one-one to network or abstract further, I think it is more ideal to abstract up further to app purpose, `expirationLedgerSeq` is the only relevant/actionable fact from an expiration right, no-one needs the keyHash, I'll change this to `expirationLedgerSeq`.

Reply:
@tsachiherman , as part of https://github.com/stellar/soroban-tools/issues/1010, wanted to confirm on expected changes to [LedgerEntryResult](https://github.com/stellar/soroban-tools/blob/main/cmd/soroban-rpc/internal/methods/get_ledger_entries.go#L21-L28), so that we can pre-plumb the same model into js client here, it will likely have `expirationLedgerSeq` and `isExpired` added correct?

Reply:
yeah and this aligns w/ RPC, exactly like you said

Reply:
yes, [discussion on #1010](https://github.com/stellar/soroban-tools/issues/1010#issuecomment-1758158987), summary was just a new `expirationLedgerSeq` will be introduced on results, so, updated to align to that model for results here in pr.